MCprep_addon/mcprep_ui.py

```diff
@@ -380,6 +380,9 @@ def update_exp_vivy_file_path(self, context):
 		env.reload_vivy_json(path)
 		env.vivy_name_changes = {}
 		env.vivy_cache = None
+	
+	def update_enable_vivy(self, context):
+		env.vivy_enabled = self.exp_vivy_material_system
```

Comment:
Just checking, this doesn't need to save any values out to work properly on next restart of Blender does it?